packages/client/src/containers/LandingPage/LandingPage.Container.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
Can you please rename this file?
We should avoid using `ComponentName.container.js` naming, since it is already in the `../container/..` folder.
You can simply use `LandingPage.js`